### ISSUE TITLE
It is compatible with esx?

### ISSUE THREAD

**Issue by hskimeros** [GHMatti/FiveM-MySQL#6]
I was wondering if this connector if I configure it and remove the mysql-async dependencies is compatible or do I have to do something else? I currently have this error:
![image](https://user-images.githubusercontent.com/29005191/45511639-d7055700-b79d-11e8-8720-448848d78d48.png)

Sorry that I opened the incident in the new connector that you are doing with node.js. I take this opportunity to ask you, better this or the node.js? and if it is the node.js, is it compatible with esx?

**Comment by GHMatti** [comment]
It should be.

If you are running a windows server you should use the mysql-async 3.0.0. It might not be better than this but it should be more consistent on reads.

If not. Notice that this folder https://github.com/GHMatti/FiveM-MySQL/tree/mysql-async-replacement still has a mysql-async folder. These contents need to be present.

Replacing mysql-async requires overwriting the MySQL.lua (which reroutes the calls to GHMattiMySQL). In your case you might either have removed too much, or you might not have the file present at all.

**Comment by hskimeros** [comment]
Thanks, I didn't really realize that great detail. I have MariaDB with debian 8. I'm going to implement it well and we'll see if it works properly this time. The only question is in resource do I have to leave the call to mysql.lua?

=)

**Comment by GHMatti** [comment]
Yes. It need to stay in the `__resource.lua`. I mean the line that says

```lua
server_script '@mysql-async/lib/MySQL.lua'
```

Well and that file needs to be present. Then all should work out.

**Comment by hskimeros** [comment]
Thank you very much for the help, I have managed with your connector to reduce 200ms in the querys.

And it already works correctly as you can see in the image:
![image](https://user-images.githubusercontent.com/29005191/45708458-d09e2300-bb81-11e8-9505-7a5d3ec28b4f.png)

I will follow your work closely and when the time comes it is possible that I change the programming of ESX to adapt it to your new connector that you are developing in node.js, a greeting!
